kie-server-parent/kie-server-wars/kie-server/src/main/assembly/assembly-ee7-container.xml

```diff
@@ -96,6 +96,7 @@
         <exclude>xml-apis:xml-apis:jar</exclude>
         <exclude>javax.mail:mail</exclude>
         <exclude>org.apache.cxf:cxf-rt-ws-policy</exclude>
+        <exclude>org.apache.cxf:cxf-rt-ws-addr</exclude>
```

Comment:
this one is to reduce number of warning when loading CXF extensions that comes from or rely on cxf-ws-policy